tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -751,8 +910,17 @@ public function test_process_directives_does_not_change_inner_html_in_math() {
 	private function evaluate( $directive_value ) {
 		$generate_state = function ( $name ) {
 			return array(
-				'key'    => $name,
-				'nested' => array( 'key' => $name . '-nested' ),
+				'key'     => $name,
+				'nested'  => array( 'key' => $name . '-nested' ),
+				'derived' => function () {
+					$state   = $this->interactivity->state();
+					$context = $this->interactivity->get_context();
```

Comment:
Do we have any tests using the global functions?

Reply:
We don't. I guess I could use them here, but I'd have to modify the global `WP_Interactivity_API` instance, which is the one used by the global functions.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6394/commits/a1fb279917ce0e2905c8d6057e33e8f7fbc3d76b.